### ISSUE TITLE
SpawnKerbals not Removed from Roster after Recovery after Contract Failure

### ISSUE THREAD

**Issue by jrossignol** [jrossignol/ContractConfigurator#564]
See https://github.com/jrossignol/ContractConfigurator/issues/443#issuecomment-242992247

**Comment by inigmatus** [comment]
begging for a milestone on this one.  GAP contract fails are essentially yielding all those delicious disappointed (surviving) tourists to the roster.  my fans are starting to get a little macabre in finding ways to get rid of them.  for the sake of Kerbals everywhere, I feel I should ping the Maker on a solution soon. :)

**Comment by jrossignol** [comment]
Yup, definitely on the list for 1.20.x.  Problem is that I'm out of town at the moment, and going to be pretty busy with KSP 1.2 until it's officially out on top of that.

Can you let me know if you have an easy repro for this in 1.2?  I assume it wasn't dependent on the g-force bug (which should be fixed now).

**Comment by inigmatus** [comment]
Pick any GAP contract with passengers, crash the plane (fail the contract), and recover survivors.  Survivors remain on roster.
